Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -28,41 +34,29 @@ spec:
   deviceClass: hdd
 ```
 
-#### Hybrid Storage Pools
+### Erasure Coded RBD Pool
 
-Hybrid storage is a combination of two different storage tiers. For example, SSD and HDD.
-This helps to improve the read performance of cluster by placing, say, 1st copy of data on the higher performance tier (SSD or NVME) and remaining replicated copies on lower cost tier (HDDs).
+This example specifies a pool with an EC 4+2 data protection scheme.  For a
+given amount of raw storage, this pool yields double the usable space compared
+to the replicated `size 3` example above.  Redundancy is provided by
+specifying an [erasure coding](#erasure-coding) profile with `codingChunks: 2`,
+akin to a conventional RAID6 volume but with much greater flexibility.  Data
+remains available if any one OSD is down and is preserved if any two are lost.
 
-**WARNING** Hybrid storage pools are likely to suffer from lower availability if a node goes down. The data across the two
-tiers may actually end up on the same node, instead of being spread across unique nodes (or failure domains) as expected.
-Instead of using hybrid pools, consider configuring [primary affinity](https://docs.ceph.com/en/latest/rados/operations/crush-map/#primary-affinity) from the toolbox.
-
-```yaml
-apiVersion: ceph.rook.io/v1
-kind: CephBlockPool
-metadata:
-  name: replicapool
-  namespace: rook-ceph
-spec:
-  failureDomain: host
-  replicated:
-    size: 3
-    hybridStorage:
-      primaryDeviceClass: ssd
-      secondaryDeviceClass: hdd
-```
-
-!!! important
-    The device classes `primaryDeviceClass` and `secondaryDeviceClass` must have at least one OSD associated with them or else the pool creation will fail.
-
-### Erasure Coded
-
-This sample will lower the overall storage capacity requirement, while also adding redundancy by using [erasure coding](#erasure-coding).
+The downside is that write performance will be significantly lower compared to
```

Comment:
With perf improvements coming in tentacle, how about we remove "significantly"? I've heard users say they see a 10-20% perf cost generally for EC, so if that's true, it doesn't seem like it's significant anyway. I like to avoid alarming wording where possible. :) 
```suggestion
The downside is that write performance will be lower compared to
```

Reply:
Re Tentacle, the scope of "fast EC" is not yet entirely clear to me, but based on a design document to which I was recently referred, I *think* those improvements are only limited to [over]writing partial stripes, i.e. RADOS objects whose size is` <= (replication * min_alloc_size)`

AIUI reads / writes of full RADOS object stripes are not affected, though I seek deeper understanding.  I also suspect at this time that these changes will not mitigate the padding space amp for very small RADOS objects, so I want to be careful to document accurately.

Reply:
Yes, definitely a lot of factors to consider. Since there are so many factors from the hardware to configuration to the workload, our approach has been to leave it as an exercise for the reader to measure in their own clusters instead of trying to document.